### PR TITLE
THX-34937 support java8

### PR DESCRIPTION
**Pull request by JustinTullgren**
Documentation update.

Points to Thunderhead Bintray package for download.

Specifies supported Android Plugin version.

Arrangement and spell/grammar check.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,95 +1,76 @@
...
-
-Apply the `aspectj` plugin:
-```groovy
-apply plugin: 'com.archinamon.aspectj'
++ Update the `allprojects` `repositories` closure to include `mavenCentral` as shown in the following example:
+``` groovy
+allprojects {
+    repositories {
+        jcenter()
+        mavenCentral()
```
Comment (ghost) [inline README.md:36]: Whatever is available on Maven Central is available on jcenter as well, so do we really need mavenCentral() here?

File: README.md [hunk 2]
```diff
@@ -1,95 +1,76 @@
...
+        google()
+    }
+}
 ```
++ Example of the **top-level** `build.gradle` file after integration:
+``` gradle
+buildscript {
+    repositories {
+        jcenter()
+        google()
```
Comment (ghost) [inline README.md:44]: `google()` should come above `jcenter()`, same with line 21 above

File: README.md (hunk 1 above)
Comment (JustinTullgren) [reply README.md:36]: It was explicitly called out in the docs before. I just formalized it. The note was that some mavenCentral dependencies for aspectj were not in jcenter. Not sure that's possible but I kept what was said and just applied it in the cleanup.

File: README.md (hunk 2 above)
Comment (JustinTullgren) [reply README.md:44]: I don't believe that is necessary here since this is a Gradle plugin. Why do you think this is a problem?

**Review by andreipop — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ghost) [reply README.md:44]: jcenter() contains artifacts dependent upon google() requiring google() first.  Gradle plugin doesn't change that dependency.

Comment (JustinTullgren) [reply README.md:44]: updated.

**Review by ghost — APPROVED** [review]

**Pull request merged**